lib/stormpath-sdk/id_site/error.rb

```diff
@@ -0,0 +1,42 @@
+module Stormpath
+  module IdSite
+    class Error < Stormpath::Error
```

Comment:
In this case we don’t receive any errors from the server. when you decode the JWT you get something like this.

 [{"iat"=>1447152616,
  "aud"=>”code”,
  "sub"=>"https://api.stormpath.com/v1/applications/hash”,
  "path"=>"",
  "state"=>"",
  "isNewSub"=>true,
  "status"=>"REGISTERED"},
 {"typ"=>"JWT", "alg"=>”algorithm”}]

for example we need to compare “aud” field from JWT with api_key_id and raise the error if they are not the same